flutter/lib/ui/geometry.dart

```diff
@@ -680,35 +680,183 @@ class Rect {
   String toString() => "Rect.fromLTRB(${left.toStringAsFixed(1)}, ${top.toStringAsFixed(1)}, ${right.toStringAsFixed(1)}, ${bottom.toStringAsFixed(1)})";
 }
 
-// A rounded rectangle with the same radii for all four corners.
+/// A radius for either circular or oval shapes.
+class Radius {
+  /// Constructs a circular radius.
```

Comment:
Add to the dartdocs that this means that the radius is used for `both [x] and [y]` or some such.